modules/custom/og_ext_cron/src/Utils/CronFunctions.php

```diff
@@ -26,6 +26,13 @@ public function clear_view_caches() {
       'pd_core_travela',
       'pd_core_travelq',
       'pd_core_wrongdoing',
+      'pd_core_ati_details.yml',
+      'pd_core_contracts_details.yml',
+      'pd_core_hospitalityq_details.yml',
+      'pd_core_reclassification_details.yml',
+      'pd_core_travelq_details.yml',
+      'pd_core_wrongdoing_details.yml',
+      'pd_core_travela_details.yml',

```

Comment:
Great! how about we remove `.yml`, the rest is the machine name for the views. Let's add `pd_core_inventory_details` in the list as well